crates/transaction-workloads-lib/src/args.rs

```diff
@@ -78,6 +78,8 @@ pub enum TransactionTypeArg {
     SmartTablePicture1MWith1KChangeExceedsLimit,
     DeserializeU256,
     SimpleScript,
+    PermissionedTransfer,
+    APTTransfer,
```

Comment:
Few things:
- Use `Apt` like in other enum variants
- How is this related to `AptFaTransfer`? Isn't it the same? I am fine having both, I think it would be nice to unify transfer under one use cases: coin, fa, permissioned, possibly veiled in the future